pkgs/development/tools/cdecrypt/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "cdecrypt";
+  version = "4.8";
+
+  src = fetchFromGitHub {
+    owner = "VitaSmith";
+    repo = "cdecrypt";
+    rev = "v${version}";
+    sha256 = "PyT60RDyp1/Co/7WHC0+KrsnrDeTJ605x1pt4OmlGYg=";
+  };
+
+  installPhase = ''
+    install -Dm755 cdecrypt $out/bin/cdecrypt
+  '';
+
+  meta = with lib; {
+    description = "A utility that decrypts Wii U NUS content files.";
+    homepage = "https://github.com/VitaSmith/cdecrypt";
+    license = licenses.gpl3;
```

Seems to be `GPLv3+` according to the [file header](https://github.com/VitaSmith/cdecrypt/blob/master/util.c#L5-L8).

```suggestion
    license = licenses.gpl3Plus;
```

---

change applied in rebased commit.